loadgen/cmd/otelbench/runner.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+)
+
+var Config struct {
+	ServerURL           *url.URL
+	SecretToken         string
+	APIKey              string
+	Secure              bool
+	Headers             map[string]string
+	CollectorConfigPath string
+}
+
+var FlagSet = flag.NewFlagSet("", flag.ExitOnError)
+
+func Init() {
+	// Server config
+	FlagSet.Func(
+		"server",
+		"server URL (default http://127.0.0.1:8200)",
+		func(server string) (err error) {
+			if server != "" {
+				Config.ServerURL, err = url.Parse(server)
+			}
+			return
+		})
+	FlagSet.StringVar(&Config.SecretToken, "secret-token", "", "secret token for APM Server")
+	FlagSet.StringVar(&Config.APIKey, "api-key", "", "API key for APM Server")
+	FlagSet.BoolVar(&Config.Secure, "secure", false, "validate the remote server TLS certificates")
```

Comment:
`insecure-skip-verify`?

Reply:
I mixed up `insecure` and `insecure-skip-verify`. Fixed.